addons/account/views/account_payment_term_views.xml

```diff
@@ -77,10 +77,10 @@
                                     Example:
                                     <field name="example_amount" class="oe_inline"/>
                                     on
-                                    <field name="example_date" class="fw-bold oe_inline"/>
+                                    <field name="example_date" class="fw-bold border-bottom example_date"/>
```

Comment:
Usually when we do a class for ourself we put "o_" before to make a difference between bootstrap and our classes, sorry i should have told you at the previous review :grimacing:

So it will be o_example_date :smile: